Libraries/LLM/Cohere.swift

```diff
@@ -145,7 +145,7 @@ public class CohereModelInner: Module {
     }
 }
 
-public class CohereModel: Module, LLMModel, KVCacheDimensionProvider {
+public class CohereModel: Module, LLMModel, KVCacheDimensionProvider,@unchecked Sendable {
```

Comment:
The models can't be `@unchecked Sendable` as they are entirely not thread safe (the MLXArrays themselves cannot be used in a multithreaded manner)

Reply:
If `@unchecked Sendable` is not used, there will be a Task warning in createModelSynchronous.

```log
Value of non-Sendable type '@isolated(any) @async @callee_guaranteed @substituted <τ_0_0> () -> @out τ_0_0 for <()>' accessed after being transferred; later accesses could race; this is an error in the Swift 6 language mode
```

<img width="1142" alt="Xcode 2024-09-10 23 21 40" src="https://github.com/user-attachments/assets/f823a666-0cbd-4fbb-94bd-1bfeca8bdf01">